pkg/v1/remote/descriptor.go

```diff
@@ -277,6 +302,55 @@ func (f *fetcher) fetchManifest(ref name.Reference, acceptable []types.MediaType
 	return manifest, &desc, nil
 }
 
+func (f *fetcher) fetchDescriptor(ref name.Reference, acceptable []types.MediaType) (*v1.Descriptor, error) {
```

Comment:
Maybe add a short comment that this is like fetchManifest but using a HEAD request instead of GET.

Reply:
Or maybe just rename to `headManifest`.